src/core/auth/client/AuthClient.ts

```diff
@@ -85,22 +87,24 @@ export class AuthClient {
     const response = await this._apiClient.login(user, password);
     if (response && typeof response === 'object' && 'token' in response) {
       await this.processAuthResponse(response);
+      // reload to init with authenticated config and profile
+      location.reload();
       return true;
```

Comment:
Do you think expired token can caught user surprised and break it's user flow?

Reply:
yes but it's expected, smoother experience will be implemented in other tasks